src/components/card/card.scss

```diff
@@ -77,6 +40,43 @@
             .col-sm-9 {
                 position: relative;
             }
+
+            .validation-message {
+                $arrow-border-width: 1rem;
+                display: block;
+                position: absolute;
+                top: 0;
+                left: 0;
+                transform: translate(16rem, 0);
+                margin-left: $arrow-border-width;
+                border: 1px solid $active-gray;
+                border-radius: 5px;
+                background-color: $ui-orange;
+                padding: 1rem;
+                max-width: 18.75rem;
+                min-height: 1rem;
+                overflow: visible;
+                color: $type-white;
+
+                &:before {
+                    display: block;
+                    position: absolute;
+                    top: 1rem;
+                    left: -$arrow-border-width / 2;
+
+                    transform: rotate(45deg);
+
+                    border-bottom: 1px solid $active-gray;
+                    border-left: 1px solid $active-gray;
+                    border-radius: 5px;
+
+                    background-color: $ui-orange;
+                    width: $arrow-border-width;
+                    height: $arrow-border-width;
+
+                    content: "";
+                }
+            }
```

Comment:
I think it would make sense to pull `.row` out of `.form`, so long as it doesn't break anything, and I don't think it should.  It would be nice if `.row` wasn't so generically named (`.form-row` would be better) — nesting kind of make sense because of that. But since we know we don't have anything else in `.card` called `.row`, I think it'd be fine/preferable to do that.

I think it'd be better to just leave `.validation-message` where it was unless you have a reason to make it more specific.  If there's another way to use `.validation-message` in `.card`, then that would be the time to make it more specific, but until then it should be more general than specific.